src/zns.ts

```diff
@@ -69,12 +69,23 @@ export default class {
     );
 
     if (!registryRecord) return null;
-    const [ownerAddress, resolverAddress] = registryRecord.arguments as [
+    let [ownerAddress, resolverAddress] = registryRecord.arguments as [
       string,
       string
     ];
     const resolution = await this.getResolverRecordsStructure(resolverAddress);
     const addresses = _.mapValues(resolution.crypto, 'address');
+    // at the moment ownerAddress is publicKey which starts with 0x 
```

Comment:
`owner` is not a public key at the moment. Old zilliqa `0x` format is not a public key: public key also starts from `0x` but it has a different length:

* Old zil address format - 40
* Public key compressed - 66
* Public key uncompressed - 128